--- a/nix/package.nix	2026-08-12 20:31:05.282393772 +0800
+++ b/nix/package.nix	2026-08-12 20:31:05.286206944 +0800
@@ -107,8 +107,8 @@
     lockFile = ../Cargo.lock;
 
     outputHashes = {
-      "smithay-0.7.0" = "sha256-V8VWa7lj8w1CP3V7H1mITD/ChlkYGAg2EW+iE/SsUzE=";
-      "smithay-drm-extras-0.1.0" = "sha256-V8VWa7lj8w1CP3V7H1mITD/ChlkYGAg2EW+iE/SsUzE=";
+      "smithay-0.7.0" = "sha256-nVdryDKVNwc7WWnKGrnZ2oQE/jETWOSruK4Rt9l6Dmg=";
+      "smithay-drm-extras-0.1.0" = "sha256-nVdryDKVNwc7WWnKGrnZ2oQE/jETWOSruK4Rt9l6Dmg=";
       "rustyscript-0.12.3" = "sha256-04yZws8aY6NpyQc0F6fg7CAwYYXer7r+eFABwafP+kU=";
     };
   };
